support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { getTimeboundCopy } from './timeBoundedCopy';
+
+describe('Time bounded copy', () => {
+  const copy = 'Hello this is a test';
+  let testCopy;
+
+  beforeEach(() => {
+    testCopy = {
+      newspaper: [
+        {
+          start: '2021-01-01',
+          end: '2021-12-31',
```

Comment:
can we call them something like firstDay and lastDay as it's not totally clear if endDate is inclusive or exclusive